components/atoms/Stagesv2.js

```diff
@@ -0,0 +1,85 @@
+import PropTypes from "prop-types";
+import { Accordion } from "../atoms/Accordion";
+import { useState, useRef } from "react";
+/**
+ *  Stages Component
+ */
+
+export default function Stages2(props) {
+  // const { items } = useContext(LanguageContext);
+  // const language = items.language;
+  // const t = language === "en" ? en : fr;
+
+  const [showData, setShowData] = useState();
+  const subJourneyData = useRef();
+
+  function onChangeHandler(option) {
+    subJourneyData.current = option.currentTarget.value;
+    props.journeys.map((journey) =>
+      journey.titleId === option.currentTarget.value
+        ? setShowData(journey.subJourney)
+        : null
+    );
+  }
+
+  return (
+    <div className="bg-circle-color p-4">
+      <div className="flex flex-col sm:flex-wrap sm:flex-row justify-center border border-black p-4 ">
+        <div className="w-full text-center">
+          <label className="text-white pr-4" htmlFor="stage">
+            Stage:{" "}
+          </label>
+
+          <select
+            className="w-auto h-8 rounded"
+            id="stage"
+            onChange={onChangeHandler}
+          >
+            <option key="0" value="0" defaultValue>
```

Comment:
If removing the buttons as per latest designs, maybe also add the `hidden` attribute here so the user can't select this option again once they've chosen a stage?

Reply:
This is probably stale now since it appears there will be an initial screen with all the stages on it as we saw this morning. Will hit resolve on this.